From bcd8c792b6cf03c9f3ae383536b0e7394d3c6c5e Mon Sep 17 00:00:00 2001
From: Jurgen Kramer <gtmkramer@xs4all.nl>
Date: Sat, 18 Feb 2017 10:48:38 +0100
Subject: [PATCH] Add native DSD support for Mutec MC3+ USB

Add native DSD support for the Mutec MC3+ USB DAC.
---
 sound/usb/quirks.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sound/usb/quirks.c b/sound/usb/quirks.c
index 01eff6c..7d47122 100644
--- a/sound/usb/quirks.c
+++ b/sound/usb/quirks.c
@@ -1357,6 +1357,7 @@ u64 snd_usb_interface_dsd_format_quirks(struct snd_usb_audio *chip,
 	case USB_ID(0x20b1, 0x2023): /* JLsounds I2SoverUSB */
 	case USB_ID(0x20b1, 0x3023): /* Aune X1S 32BIT/384 DSD DAC */
 	case USB_ID(0x2616, 0x0106): /* PS Audio NuWave DAC */
+	case USB_ID(0x29a2, 0x0086): /* Mutec MC3+ USB */
 		if (fp->altsetting == 3)
 			return SNDRV_PCM_FMTBIT_DSD_U32_BE;
 		break;
-- 
2.9.3

